src/GitHub.VisualStudio/Helpers/ActiveDocumentSnapshot.cs

```diff
@@ -19,7 +19,7 @@ public class ActiveDocumentSnapshot : IActiveDocumentSnapshot
         public ActiveDocumentSnapshot([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
         {
             StartLine = EndLine = -1;
-            Name = Services.Dte2?.ActiveDocument?.FullName;
+            Name = Services.Dte2?.ActiveDocument?.ProjectItem.Name;
```

Comment:
I gave this a try by making a new solution without any projects and added a .txt file and `ProjectItem` still exists. VS might be considering a solution a project in the case when no project exists, or still considers it a project if it does.